src/material-examples/text-field-autosize-basic/text-field-autosize-basic-example.html

```diff
@@ -0,0 +1,11 @@
+<mat-form-field>
+  <mat-label>Autosize textarea basic example</mat-label>
+  <textarea matInput cdkTextareaAutosize></textarea>
+</mat-form-field>
+<mat-form-field>
```

Comment:
I don't know how consistent we've been with it, but in general we should avoid using Material components in CDK examples.

Reply:
Ok @crisbeto how do you recommend I proceed? The other examples seem to use Material Components, so unsure how to go about this.
i.e a basic textarea with placeholder and `cdkTextAreaAutosize` directive added?

Also then I will apply this change to all text-field autosize examples

Reply:
Yes, it should just be a plain `textarea` without any Angular Material components

Reply:
Changed

Reply:
Did you push your changes? The code still has `mat-form-field`

Reply:
Pushed.
 All `mat` component usage removed ..@jelbourn - I don't know how this will look though considering the other CDK components have mat styling. 

What it should look like now:
![image](https://user-images.githubusercontent.com/1691165/60992258-6b076700-a34c-11e9-9568-555b1215ed8d.png)